doc/releases/release-notes-3.5.rst

```diff
@@ -475,6 +475,9 @@ Drivers and Sensors
 
 * Disk
 
+  * Ramdisk driver is now configured using devicetree, and supports multiple
+    instances
```

Comment:
does this impact users to the point where entries should be added to the migration guide? Ex. Kconfig options that were removed and replaced by binding properties?

Reply:
Yes, probably so- will make a PR for this